pd/src/staking.rs

```diff
@@ -3,6 +3,17 @@ use serde::{Deserialize, Serialize};
 use tendermint::{vote, PublicKey};
 
 const PENUMBRA_BECH32_VALIDATOR_PREFIX: &str = "penumbravalpub";
+
+/// A destination for a portion of a validator's commission of staking rewards.
+#[derive(Debug, Deserialize, Serialize, PartialEq, Eq, Clone)]
+pub struct FundingStream {
+    /// The destinatination address for the funding stream..
+    pub address: Address,
+
+    /// The portion (in terms of basis points) of the validator's total commission that goes to
+    /// this funding stream.
+    pub rate_bps: u16,
```

Comment:
If we specify each funding stream as a percentage of the validator's total commission, we have an extra invariant that the sum of the `rate_bps` must be 10,000.

Could we instead specify the rate for each funding stream in absolute terms, and let the validator's total commission be specified implicitly as the sum of the commission for each funding stream?

Reply:
(Alternatively, if that's what this is already supposed to be doing -- the total rate disappeared from the `Validator` structure, so maybe it is? -- I guess the comment should be changed to say that it's a portion of the total rewards for that validator's delegation pool.